AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_entity_utils.py

```diff
@@ -21,14 +22,25 @@
 from editor_python_test_tools.utils import Report
 
 
+class Entity_Type(Enum):
```

Comment:
nit: PEP8 states UpperCamelCase instead of Upper_Snake_CamelCase for class names

Reply:
I wanted to avoid confusion with azlmbr.entity.EnityType(). The pycharm IDE refactor rename was colliding with instances of that and I wanted a clearly different name.

Reply:
EditorEntityType may also work

Reply:
EditorEntityType it is